### ISSUE TITLE
Can't locate Dist/Build.pm in @INC

### ISSUE THREAD

**Issue by eserte** [skaji/perl-File-Copy-clonefile#3]
Fails to build on all of my CPAN.pm-based smokers like this:
```
Output from '/usr/perl5.38.2Dp/bin/perl Build.PL':

Can't locate Dist/Build.pm in @INC (you may need to install the Dist::Build module) (@INC entries checked: /var/tmp/cpansmoker-1023/2024071712/CPAN-Reporter-lib-1Wz9 /home/cpansand/.cpan/build/2024071712/ExtUtils-Builder-0.008-0/blib/arch /home/cpansand/.cpan/build/2024071712/ExtUtils-Builder-0.008-0/blib/lib /usr/perl5.38.2Dp/lib/site_perl/5.38.2/amd64-freebsd-ld /usr/perl5.38.2Dp/lib/site_perl/5.38.2 /usr/perl5.38.2Dp/lib/5.38.2/amd64-freebsd-ld /usr/perl5.38.2Dp/lib/5.38.2 .) at Build.PL line 7.
BEGIN failed--compilation aborted at Build.PL line 7.
```
Probably a missing configure_requires declaration?

**Comment by skaji** [comment]
Thanks for reporting.
Configure requires are declared.
https://metacpan.org/release/SKAJI/File-Copy-clonefile-v0.0.5/source/META.json#L20

**Comment by skaji** [comment]
ping?

**Comment by eserte** [comment]
Strange, it does not seem to work. The matrix is full of UNKNOWN reports: http://fast-matrix.cpantesters.org/?dist=File-Copy-clonefile%200.0.5

**Comment by eserte** [comment]
OK, I read now that probably only macos is supported. However, I would then expect that the reports should show NA (not available) instead of UNKNOWN (happens mostly due to configuration failures). Sadly cpantesters is currently down, so we cannot look at the individual test reports.

**Comment by eserte** [comment]
I think I tracked it down to two bugs:
* Dist::Build had an unsufficient dependency in 0.006, which is fixed in 0.007: https://metacpan.org/release/LEONT/Dist-Build-0.007/source/Changes#L4
* CPAN.pm seems to generate a test report even if the build of a configure_requires fails (no report so far)

So probably no problem in this module of yours (and some others like App-RemoteCommand-0.991 were also affected). Closing.
